lib/common/api/callbacks-registry.js

```diff
@@ -10,26 +10,25 @@ class CallbacksRegistry {
 
   add (callback) {
     // The callback is already added.
-    var filenameAndLine, id, location, match, ref, regexp, stackString
-    id = v8Util.getHiddenValue(callback, 'callbackId')
-    if (id != null) {
-      return id
-    }
-    id = ++this.nextId
+    let id = v8Util.getHiddenValue(callback, 'callbackId')
+    if (id != null) return id
+
+    id = this.nextId += 1
 
     // Capture the location of the function and put it in the ID string,
     // so that release errors can be tracked down easily.
-    regexp = /at (.*)/gi
-    stackString = (new Error()).stack
+    const regexp = /at (.*)/gi
+    const stackString = (new Error()).stack
+
+    let filenameAndLine
+    let match
+
     while ((match = regexp.exec(stackString)) !== null) {
-      location = match[1]
-      if (location.indexOf('(native)') !== -1) {
-        continue
-      }
-      if (location.indexOf('electron.asar') !== -1) {
-        continue
-      }
-      ref = /([^/^)]*)\)?$/gi.exec(location)
+      const location = match[1]
+      if (location.includes('native')) continue
```

Comment:
> `.includes('native')` 

In the removed code it's `.indexOf('(native)')`, surrounded by braces.
I'll fix it in the Ch61 branch, there are merge conflicts.